src/main/com/example010/radek/v2/Soldier.java

```diff
@@ -0,0 +1,60 @@
+package main.com.example010.radek.v2;
+
+public class Soldier implements BaseFighter {
+

```

Comment:
soldier nebude implementovat BaseFighter, muzes smazat, asi chyba v zadani

Reply:
Hotovo, snad je to takto správně a pochopili jsme se

Reply:
ne, smazat ty metody, attack defense a running